### PR TITLE
set up IBM Watson tone analyzer

### PR DESCRIPTION
**Pull request by jhimitu**

### CODE REVIEW

File: tone_analyzer/toneAnalyzer.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  version: process.env.TONE_ANALYZER_VERSION_DATE,
+  iam_apikey: process.env.TONE_ANALYZER_APIKEY,
+});
+
+const text = 'Great stuff';
+const toneParams = {
+  tone_input: { text },
+  content_type: 'application/json',
+};
+toneAnalyzer.tone(toneParams, (error, toneAnalysis) => {
```
Comment (joanneskim) [inline tone_analyzer/toneAnalyzer.js]: What does this do?

File: package.json [hunk 2]
```diff
@@ -29,7 +29,8 @@
     "react-apollo": "^2.2.2",
     "react-dom": "^16.5.2",
     "sequelize": "^4.39.0",
-    "sequelize-cli": "^4.1.1"
+    "sequelize-cli": "^4.1.1",
+    "watson-developer-cloud": "^3.11.1"
```
Comment (Skilledmonkey) [inline package.json]: gettting the Watson done!

Comment (joanneskim) [reply package.json]: niceee

File: tone_analyzer/toneAnalyzer.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+require('dotenv').config();
+const ToneAnalyzerV3 = require('watson-developer-cloud/tone-analyzer/v3');
+
+const toneAnalyzer = new ToneAnalyzerV3({
+  url: process.env.TONE_ANALYZER_URL,
+  version: process.env.TONE_ANALYZER_VERSION_DATE,
+  iam_apikey: process.env.TONE_ANALYZER_APIKEY,
+});
+
+const text = 'Great stuff';
```
Comment (Skilledmonkey) [inline tone_analyzer/toneAnalyzer.js]: it is great!

File: tone_analyzer/toneAnalyzer.js (hunk 1 above)
Comment (jhimitu) [reply tone_analyzer/toneAnalyzer.js]: The object is setting the parameters to be passed into the tone method. tone_input specifies the text to be analyzed and content_type specifies the type.

**Pull request merged**